hoverctl/wrapper/simulation.go

```diff
@@ -38,14 +37,27 @@ func ExportSimulation(target configuration.Target, urlPattern string) ([]byte, e
 		return nil, errors.New("Could not export from Hoverfly")
 	}
 
-	var jsonBytes bytes.Buffer
-	err = json.Indent(&jsonBytes, body, "", "\t")
-	if err != nil {
+	var view v2.SimulationViewV6
+	if err := json.Unmarshal(body, &view); err != nil {
 		log.Debug(err.Error())
-		return nil, errors.New("Could not export from Hoverfly")
+		return nil, err
+	}
```

Comment:
you can read response body into a struct in one go: 
```
err := json.NewDecoder(response.Body).Decode(&view)
```

Reply:
true